build/Common.nonprod.props

```diff
@@ -25,10 +25,14 @@
     <BenchmarkDotNetPkgVer>[0.12.1,0.13)</BenchmarkDotNetPkgVer>
     <CommandLineParserPkgVer>[2.3.0,3.0)</CommandLineParserPkgVer>
     <DotNetXUnitCliVer>[2.3.1,3.0)</DotNetXUnitCliVer>
-    <GoogleProtobufPkgVer>[3.13.0,4.0)</GoogleProtobufPkgVer>
+	<GoogleProtobufPkgVer>[3.13.0,4.0)</GoogleProtobufPkgVer>
```

Comment:
typo? (TAB)

Reply:
Yup thanks for catching that, sorry you have to keep reviewing this I will be more careful with commits.